### ISSUE TITLE
[Feature request] CI: ignore push to master of forked repo

### ISSUE THREAD

**Issue by jannick0** [msys2/MSYS2-packages#2258]
Pushing `master` to a repo forked from this repo triggers a full blown CI run which is unneccessary I think.  Would it be possible to make CI ignoring CI runs when `master` changes?

This would make it possible to keep `master` of the forked repo on github in sync with this repo.  The same would apply to the MINGW packages repo, of course. 

Thanks.

**Comment by Biswa96** [comment]
Why not disable the GitHub Actions in your forked repository? GitHub now does not run CI in forked repository unless it is enabled.

**Comment by jannick0** [comment]
I believe pushers are definitely not wanted to run their tests in the upstream PR folder, do they? ;)  Good luck with the clutter if so. :)

I prefer to run the tests in the fork first, then, once ready, I create the PR. This is better I think  ... or can I config in my fork that CI ignores `master`?

**Comment by Biswa96** [comment]
Agreed, you have a valid point. Generally I do not use GHA in my forked repository because I do not get any faster compilation time than my PC. So, I disabled GHA and compile packages in my PC before a pull request. But hey, choices differ 😃
